src/rp2_common/pico_runtime/CMakeLists.txt

```diff
@@ -68,6 +68,7 @@ endif()
 # FPGA_CHECK            - checks for FPGA which allows Raspberry Pi to run your binary on FPGA;
 # PANIC                 - default panic impl which brings in stdio;
 # AUTO_INIT_MUTEX       - auto init mutexes, without this you get no printf mutex either;
+# CRT0_FAR_CALLS        - use blx not bl far calls from crt0
```

Comment:
Is the "far calls from crt0" here supposed to be "for calls from crt0" ?

Reply:
yes